public/achievementInfo.php

```diff
@@ -231,11 +249,7 @@ function updateAchievementTypeFlag(typeFlag) {
             echo "<div class='devbox mb-3'>";
             echo "<span onclick=\"$('#resetboxcontent').toggle(); return false;\">Reset Progress ▼</span>";
             echo "<div id='resetboxcontent' style='display: none'>";
-            echo "<form action='/request/user/reset-achievements.php' method='post' onsubmit='return confirm(\"Are you sure you want to reset this progress?\")'>";
-            echo csrf_field();
-            echo "<input type='hidden' name='a' value='$achievementID'>";
```

Comment:
looks like the input name was the culprit there

Reply:
Yep. But after fixing that, the submission would page turn to the javascript response, which is why I replaced the form with an ajax call.

Reply:
oh, i see. disregard my other comment then. this mix of json and default form requests is a mess 😅